### PR TITLE
fix gateway_ip description

### PR DESCRIPTION
**Pull request by morozovalekseywot**

### CODE REVIEW

File: vkcs/networking/resource_subnet.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func ResourceNetworkingSubnet() *schema.Resource {
 				Optional:      true,
 				ForceNew:      false,
 				Computed:      true,
-				Description:   "Default gateway used by devices in this subnet. Leaving this blank and not setting `no_gateway` will cause a default gateway of `.1` to be used. Changing this updates the gateway IP of the existing subnet.",
+				Description:   "Default gateway used by devices in this subnet. Leaving this blank and not setting `no_gateway` will cause a default gateway of `.1` to be used for the Neutron SDN, but the gateway does not have a default value for the Sprut SDN. Changing this updates the gateway IP of the existing subnet.",
```
Comment (paaanic) [inline vkcs/networking/resource_subnet.go:118]:
```suggestion
				Description:   "Default gateway used by devices in this subnet. Leaving this blank and not setting `no_gateway` will cause a default value to be used. Changing this updates the gateway IP of the existing subnet.",
```

Comment (paaanic) [reply vkcs/networking/resource_subnet.go:118]: To be mentioned, if I remember correctly, we cannot edit gateway IP in Sprut SDN.

Comment (morozovalekseywot) [reply vkcs/networking/resource_subnet.go]: fix

**Comment by paaanic** [comment]
Add a changelog entry.

**Review by paaanic — APPROVED** [review]

**Pull request merged**